extensions/blocks/calendly/index.js

```diff
@@ -40,15 +50,15 @@ export const settings = {
 		html: false,
 	},
 	edit,
-	save: ( { attributes: { url } } ) => <a href={ url }>{ url }</a>,
+	save,
 	attributes,
 	example: {
 		attributes: {
-			submitButtonText: __( 'Schedule time with me', 'jetpack' ),
 			hideEventTypeDetails: false,
 			style: 'inline',
 			url: CALENDLY_EXAMPLE_URL,
 		},
+		innerBlocks: [ innerButtonBlock ],
```

Comment:
The example is `style: 'inline'`, which is the actual widget embed, where the Button is not displayed.
So, defining the `innerBlocks` here is unnecessary.

This would also slightly simplify the how we pass around data.
We can avoid definining and exporting `innerButtonBlock` here in the index.js, and instead use it straight away in edit.js.

Reply:
Good catch. I've cleaned that up and moved it to edit.js.